examples/quickstarts/cakephp-mysql-persistent.json

```diff
@@ -60,7 +60,10 @@
       "kind": "Route",
       "apiVersion": "v1",
       "metadata": {
-        "name": "${NAME}"
+        "name": "${NAME}",
+        "annotations": {
+          "template.openshift.io/expose": ""
+        }
```

Comment:
OK, I'm going to leave these changes in the PR for now as one of them is required for the extended test to work.  After api-review and before merge I'll send PRs to the upstream repos and run hack/update-external-examples.sh.  Can't wait.